housekeeping-vacuum-tool/README.md

```diff
@@ -60,11 +60,11 @@ The `dry-run` option allows you to observe the status of paths on the file syste
 |`tables.table-name`|Yes| The Hive table name for the table the vacuum tool will interrogate.|
 |`housekeeping.schema-name`|No|The schema name that is used in your housekeeping instance. Defaults to `housekeeping`.|
 |`housekeeping.h2.home`|No|The path to your H2 filesystem database specified as a full path. Defaults to `$HOME/data`.|
-|`housekeeping.datasource.driver-class-name` |No|The fully qualified class name of your database driver. Defaults to the H2 driver if not configured.|
-|`housekeeping.datasource.url` |No| JDBC URL for your database. Defaults to H2 filesystem database if not specified. |
-|`housekeeping.datasource.username` |No| Username for your database.|
-|`housekeeping.datasource.password` |No| Password for your database.|
-|`housekeeping.db-init-script`|No|A file containing a script to initialise your schema can be provided if it does not already exist. Defaults to `classpath:/schema.sql`.|
+|`housekeeping.data-source.driver-class-name` |No|The fully qualified class name of your database driver. Defaults to the H2 driver if not configured.|
```

Comment:
did we change the name of the datasoure property? (to data-source)

Reply:
This was resolved, but for the next person reviewing this: 
I didn't change the code for the key name. I just changed the docs so the key name is the same as the one in `houskeeping-parent` docs :)